network/src/connectivity_manager/mod.rs

```diff
@@ -483,6 +483,17 @@ where
             self.network_context
         );
 
+        // Log the eligible peers state periodically so we see what our current state is
+        sample!(SampleRate::Duration(Duration::from_secs(60)), {
```

Comment:
We can probably ramp the sample rate down to every 5 minutes or something

Reply:
I'd rather err on logging more often than less for now.  Especially since I removed the preprocessing, it should be much more lightweight.